chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -669,6 +670,7 @@ maxLength.apiDocumentationLink=1024
 maxLength.exportDocumentationLink=1024
 maxLength.documentationUrlLink=1024
 maxLength.useCasesLink=1024
+maxLength.serviceBaseUrlListLink=1024
```

Comment:
Need to check this in a reviewer somewhere (I can't find it in the PR but if I missed the file just tell me!). There is a max length reviewer that checks a number of other fields.

Reply:
I can't find where any of the other criteria URL max lengths are being checked either. Neither of the FieldLengthReviewer.java files ("regular" and "pending") check criteria fields, so I'm not sure where these should be checked either